generate-certs

```diff
@@ -8,7 +8,7 @@ echo
 export CA_KEY=${CA_KEY-"ca-key.pem"}
 export CA_CERT=${CA_CERT-"ca.pem"}
 export CA_SUBJECT=${CA_SUBJECT:-"test-ca"}
-export CA_EXPIRE=${EXPIRE:-"60"}
+export CA_EXPIRE=${CA_EXPIRE:-"60"}
```

Comment:
I am +1 for keeping the ENV vars in a consistent nomenclature.  Even though i'm only community this is +1 LGTM